pkg/models/infra.go

```diff
@@ -12,13 +22,48 @@ func (p *Infrastructure) AutoFill() {
 	if p.Directory == "" {
 		p.Directory = "./" + p.ProjectModuleName
 	}
+	logger := pkg.GetLogger()
+	goModPath, err := utils.FindPathFromFile(p.Directory, "go.mod")
+	if err != nil {
+		logger.Fatal("go.mod file not found", "err", err)
+	}
+	p.Directory = goModPath
+	goMod, err := GetModuleNameFromGoModFile(goModPath)
+	if err != nil {
+		logger.Fatal("couldn't get module name from go.mod", "err", err)
+	}
+	p.ProjectModuleName = goMod.Module
```

Comment:
If fields are not provided from the command flags then it should be autofilled. 
Can you modify your implementation in such way? I think it's better to give users the option to override the autofilled items if they require.